website/docs/gitops-templates/capd-example.mdx

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
same here, it doesn't appear in the left nav.

Reply:
Ok, I think I know why I thought they weren't there: The file lives in ./gitops-templates/ but in the nav it's actually a level further down the hierarchy. Shouldn't the file locations represent the hierarchy?

Reply:
They are required to represent the hierarchy if you are autogenerating the sidebars. But if you are not then it is optional.

I personally don't like mirroring it exactly, beyond grouping to a point, because it makes it easy to rearrange where they appear later on by just moving lines in the `sidebar.js`. BUT I will not be the one maintaining this, so will move if that is preferred.

Reply:
I'm fine either way. If it irritates someone in the future we can always change it.